### PR TITLE
[name] make addMultilingualNames() add mac names by default

### PR DESCRIPTION
**Pull request by justvanrossum**
As discussed in #930.

### CODE REVIEW

File: Lib/fontTools/ttLib/tables/_n_a_m_e.py [hunk 1]
```diff
@@ -184,15 +188,16 @@ def addMultilingualName(self, names, ttFont=None, nameID=None):
...
-			else:
+			if windows:
+				windowsName = _makeWindowsName(name, nameID, lang)
+				if windowsName is not None:
+					self.names.append(windowsName)
+				else:
+					# We cannot not make a Windows name: make sure we add a
+					# Mac name as a fallback. This can happen for exotic
+					# BCP47 language tags that have no Windows language code.
+					mac = True
```
Comment (anthrotype) [inline Lib/fontTools/ttLib/tables/_n_a_m_e.py]: what if there is neither windows nor mac compatible name? shouldn't we error, or return None or something?

**Comment by justvanrossum** [comment]
Hm, not sure why the tests are failing, maybe I messed something up locally when merging #1357

File: Lib/fontTools/ttLib/tables/_n_a_m_e.py (hunk 1 above)
Comment (justvanrossum) [reply Lib/fontTools/ttLib/tables/_n_a_m_e.py]: Good question. In the previous revision this situation passed silently also. Leaning towards an error.

**Comment by anthrotype** [comment]
@justvanrossum what's the status of this? Can it be merged?

this TODO in varLib may be related:
https://github.com/fonttools/fonttools/blob/1fd9bf6c298c7aaceb7b87bb0d4b71041f2af740/Lib/fontTools/varLib/__init__.py#L78-L83

**Comment by justvanrossum** [comment]
Can be merged as far as I'm concerned, thanks!

**Pull request merged**